substrate/frame/revive/fixtures/build.rs

```diff
@@ -183,13 +183,8 @@ mod build {
 		let fixtures_dir: PathBuf = env::var("CARGO_MANIFEST_DIR")?.into();
 		let contracts_dir = fixtures_dir.join("contracts");
 		let uapi_dir = fixtures_dir.parent().expect("uapi dir exits; qed").join("uapi");
```

Comment:
This is also wrong because it expects the uapi crate being co-located with the fixtures dir? Which is not the case when building outside the mono repo.

Reply:
ah right added a test to check if the dir exists